system-int-test/test/cmr/system_int_test/search/collection_shapefile_simplification_search_test.clj

```diff
@@ -0,0 +1,180 @@
+
+        outer-cart (umm-s/ords->ring -5.26 -22.59 11.56 -22.77 10.47 -11.29 -5.86 -11.37 -5.26 -22.59)
+        hole1-cart (umm-s/ords->ring 6.95 -17.95 2.98 -17.94 3.92 -20.08 6.95 -17.95)
+        hole2-cart (umm-s/ords->ring 5.18 -13.08 -1.79 -12.99 -2.65 -15 4.29 -14.95 5.18 -13.08)
+        polygon-with-holes-cart (make-coll :cartesian "polygon-with-holes-cart" (poly/polygon [outer-cart hole1-cart hole2-cart]))
+
+        ;; Points
+        washington-dc (make-coll :geodetic "washington-dc" (p/point -77 38.9))
+        richmond (make-coll :geodetic "richmond" (p/point -77.4 37.54))
+        north-pole (make-coll :geodetic "north-pole" (p/point 0 90))
+        south-pole (make-coll :geodetic "south-pole" (p/point 0 -90))
+        am-point (make-coll :geodetic "am-point" (p/point 180 22))
+        esri-point (make-coll :geodetic "esri-point" (p/point -80 35))]
+    (index/wait-until-indexed)
+
+    (doseq [fmt (keys formats)
+            :let [{extension :extension mime-type :mime-type} (get formats fmt)]]
+      (testing (format "Search by %s shapefile with simplification" fmt)
+        (are3 [shapefile point-counts items]
+              (let [found (search/find-refs-with-multi-part-form-post
+                           :collection
+                           [{:name "shapefile"
+                             :content (io/file (io/resource (str "shapefiles/" shapefile "." extension)))
+                             :mime-type mime-type}
+                            {:name "simplify-shapefile"
+                             :content "true"}
+                            {:name "provider"
+                             :content "PROV1"}])
+                    [original-point-count new-point-count] point-counts
+                    headers (:headers found)]
+                (is (= {"original-point-count" original-point-count "new-point-count" new-point-count}
+                       (json/parse-string (get headers "CMR-Shapfile-Simplification"))))
+                (d/assert-refs-match items found))
+
+              "Polygons that are already simple enough are not reduced further"
+              "box"
+              [5 5]
+              [whole-world very-wide-cart washington-dc richmond]
+
+              "Many Points Polygons South America"
+              "south_america"
+              [49876 3487]
+              [wide-south-cart wide-south whole-world]
+
+              "Many Points Polygons South America with Hole"
+              "south_america_with_hole"
+              [49881 3492]
+              [wide-south-cart wide-south whole-world]
+
+              "Many Point Polygons Africa"
+              "africa"
+              [12861 2257]
+              [normal-line very-wide-cart wide-south-cart wide-south polygon-with-holes whole-world normal-brs normal-line-cart])))
+
+    ;; ESRI must be tested separately fromn other formats, because it is simplified down to slightly
```

Comment:
I assume you've verified that the shape looks correct, but just has slightly fewer points.

Reply:
Correct.